### ISSUE TITLE
SQFormScrollableCard Content area styles

### ISSUE THREAD

**Issue by jamesgriffith9519** [SelectQuoteLabs/SQForm#441]
As we continue to use the SQFormScrollableCard in new ways, we have noticed that when your content inside of the form is a little more complex than simply form fields, you may want or need the ability to adjust the parent container styling to better accommodate.  We ran into this issue specifically today while working on a new view that essentially disables the text header using the new IsHeaderDisabled prop, and then consumes some new custom components that have their own custom headers.

The issue is that if we look at the SQFormScrollableCard code, We can see that the "CardContent" parent container has some styling that is hard set and not accessable. In our case the specific issue is the padding that is set here:

 ` cardContent: {
      gridArea: 'content',
      overflowY: 'auto',
      padding: ${theme.spacing(2)}px
    },`

This styling sets padding of 16px for the scrollable content area, and in our case, we have custom headers that are "sticky" headers and they stay at the top of the scrollable area, however, that padding causes content to clip behind the headers and makes for an eyesore in the UI. 

TLDR: an enhancement idea would be to expose a prop that would allow us to adjust the styling properties of our scrollable parent container to better accommodate forms that are utilizing more complex/custom content. 

I've created a 3-4ish minute loom demonstrating this issue to allow for a bit of a visual. https://www.loom.com/share/a54ffeb4cc104a27811588834df0218e

**Comment by laurelbean** [comment]
@jamesgriffith9519 Could you check if this is still an issue after upgrading our components to use `sx` props?
